platform/android/os_android.cpp

```diff
@@ -387,6 +387,33 @@ void OS_Android::process_touch(int p_what, int p_pointer, const Vector<TouchPos>
 
 			ERR_FAIL_COND(touch.size() != p_points.size());
 
+			if (touch.size() == 1) {
+				Point2 d = (p_points[0].pos - touch.write[0].pos);
+				if (d.x > 5 || d.x < -5 || d.y > 5 || d.y < -5) {
```

Comment:
Can you move the value `5` into a constant and add comments describing its purpose.